src/management/api/analytics/logs/log.component.ts

```diff
@@ -16,60 +16,108 @@ import NotificationService from '../../../../services/notification.service';
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-const LogComponent: ng.IComponentOptions = {
-  bindings: {
-    log: '<'
-  },
-  controller: function($state: StateService, NotificationService: NotificationService, Constants: any) {
-    'ngInject';
-    this.Constants = Constants;
-    this.NotificationService = NotificationService;
 
+interface IQueryParam {
+  key: string;
+  value: string;
+}
+
+class LogComponentController {
+  public log: {
+    clientRequest: any
+    proxyRequest: any
+    clientResponse: any
+    proxyResponse: any
+  };
+  public backStateParams: { q: any; size: any; from: any; to: any; page: any };
```

Comment:
not a big fan of this inline formatting, why previous object is on multiple lines ?